src/Umbraco.Web.UI/Umbraco.Web.UI.csproj

```diff
@@ -74,6 +74,7 @@
     <PackageReference Include="Umbraco.Code" Version="1.2.0">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Umbraco.TheStarterKit" Version="9.0.0" />
```

Comment:
Was this intended to push to source control?

Reply:
Nope, thanks

Reply:
```suggestion

```